pkg/controllers/applicationset_controller.go

```diff
@@ -438,7 +445,16 @@ func (r *ApplicationSetReconciler) createOrUpdateInCluster(ctx context.Context,
 		action, err := utils.CreateOrUpdate(ctx, r.Client, found, func() error {
 			// Copy only the Application/ObjectMeta fields that are significant, from the generatedApp
 			found.Spec = generatedApp.Spec
+
+			// Preserve argo cd notifications state
```

Comment:
Tiny tweak:
Can you change this line to
```
// Preserve argo cd notifications state (https://github.com/argoproj-labs/applicationset/issues/180)
```
(ie appending the issue # for some context)

Reply:
done :+1: